package.json

```diff
@@ -51,6 +51,7 @@
     "webpack-sources": "^1.0"
   },
   "devDependencies": {
+    "@texastribune/ds-toolbox-tasks": "^0.0.2",
```

Comment:
I think this will have to be a `dependency`, not a `devDependency`, because Heroku only installs packages listed in the former when this app is deployed.